### PR TITLE
KTOR-7764 Extract messages for localization and proofreading

### PR DESCRIPTION
**Pull request by Stexxe**
https://youtrack.jetbrains.com/issue/KTOR-7764

### CODE REVIEW

**Review by e5l — APPROVED** [review]

**Comment by e5l** [comment]
nicely done

File: internal/app/i18n/bundle.en.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	JdkServerDownloadError:            "An error occurred while downloading from a JDK server. Please try again later.\n",
+	ChecksumVerificationFailed:        "Checksum verification for the downloaded JDK failed",
+	UnableMakeFileExec:                "Unable to make %s file executable.\n",
+	UnexpectedError:                   "Unexpected error occurred.\n",
+	UnexpectedErrorWithArg:            "Unexpected error %s\n",
+	UnableCreateStoreJdkDir:           "Unable to create a root directory %s to store downloaded JDKs.\n",
+	UnrecognizedFlagsError:            "Unrecognized flags: %s\n",
+	NoCommandError:                    "Expected a command",
+	CommandNotFoundError:              "Command '%s' not found\n",
+	CommandArgumentsError:             "Expected %d argument[s]: %s for the %s command\n",
```
Comment (vnikolova) [inline internal/app/i18n/bundle.en.go]: I'd suggest rewording this to `"Command %s requires %d argument[s]: %s.\n",`

File: internal/app/i18n/bundle.en.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	JavaHomeJdkIdeaInstruction:        "You can also set the JAVA_HOME environment variable permanently or add the following JDK in the IntelliJ IDEA: \n",
+	ToolSummary:                       "Ktor is a tool mainly for generating Ktor projects.\n\n",
+	OptionsCaption:                    "The options are:",
+	CommandsCaption:                   "The commands are:",
+	VerifyingJdk:                      "Verifying %s...\n",
+	CreatingDir:                       "Creating directory %s\n",
+	Extracting:                        "Extracting %s to %s\n",
+	RequestGenServer:                  "Requesting generation server...",
+	ExtractingArchiveToDir:            "Extracting downloaded archive to directory %s\n",
+	ExtractProjectArchive:             "Extracting project archive... ",
```
Comment (vnikolova) [inline internal/app/i18n/bundle.en.go:47]: Is the empty space in the end on purpose? It also appears on lines 61,63 and 67

File: internal/app/i18n/bundle.en.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	ProjectExtractError:               "Failed to extract downloaded archive to the directory %s.\n",
+	JdkExtractError:                   "Failed to extract downloaded JDK to the directory %s.\n",
+	DirAlreadyExist:                   "The directory already exists.",
+	UnableExtractJdk:                  "Failed to extract downloaded JDK.\n",
+	UnableDownloadJdk:                 "Failed to download JDK %s for %s %s\n",
+	JdkServerError:                    "Error connecting to the JDK server. Please try again later.\n",
+	JdkServerDownloadError:            "Error downloading from the JDK server. Please try again later.\n",
+	ChecksumVerificationFailed:        "Checksum verification failed for the downloaded JDK.",
+	UnableMakeFileExec:                "Failed to make %s executable.\n",
+	UnexpectedError:                   "An unexpected error occurred.\n",
```
Comment (vnikolova) [inline internal/app/i18n/bundle.en.go:30]: This isn't a very helpful message. It would be nice to provide some pointers how to solve it.

**Review by vnikolova — COMMENTED** [review]
Neat! I took liberty of adding punctuation and rewording the messages slightly to keep them concise. It might be nice to add suggestions of how to solve the errors where the reasons are not clear.

File: internal/app/i18n/bundle.en.go (hunk 2 above)
Comment (Stexxe) [reply internal/app/i18n/bundle.en.go:47]: Yes

File: internal/app/i18n/bundle.en.go (hunk 3 above)
Comment (Stexxe) [reply internal/app/i18n/bundle.en.go:30]: After this message, the next message says to read the log file for the error description. Generally, if the user encounters this error, it means a bug in the application.

Comment (vnikolova) [reply internal/app/i18n/bundle.en.go:30]: Then it's good!
